Client/mods/deathmatch/logic/luadefs/CLuaPedDefs.cpp

```diff
@@ -1811,6 +1813,43 @@ int CLuaPedDefs::SetPedAimTarget ( lua_State* luaVM )
 }
 
 
+int CLuaPedDefs::SetPedStat ( lua_State* luaVM )
+{
+    // Verify the argument
+    CClientEntity* pEntity = NULL;
+    unsigned short usStat = 0;
+    float fValue = 0;
+    CScriptArgReader argStream ( luaVM );
+    argStream.ReadUserData ( pEntity );
+    argStream.ReadNumber ( usStat );
+    argStream.ReadNumber ( fValue );
+
+    if ( !argStream.HasErrors ( ) )
```

Comment:
This shouldn't check whether the argstream has an error 4 times in a row. Putting all of it into one if-else-statement would improve readability.